src/components/Loader.js

```diff
@@ -0,0 +1,12 @@
+import loader from '../assets/call-to-action.png';
+
+const Loader = () =>{
+    return(
+        <div className="loader-container">
+            <img src={loader} className="loader" alt="spinning loader"/>
```

Comment:
Very nitpicky but could maybe make the alt description a little more specific for accessibility.